apimocks/datagrid/tiles.json

```diff
@@ -0,0 +1,74 @@
+{
+    "_links": {
+        "self": {
+            "href": "http://husky.lo:7878/admin/api/datagrid/tiles"
+        }
+    },
+    "_embedded": {
+        "items": [
+            {
+                "id": "1",
+                "title": "Folder title 1",
+                "elements": 100
```

Comment:
In other APIs we call this field `total`, indicates to me better that this is a number.